samples/colorpicker/src/com/amazon/ask/colorpicker/handlers/LaunchRequestHandler.java

```diff
@@ -30,7 +30,8 @@ public boolean canHandle(HandlerInput input) {
 
     @Override
     public Optional<Response> handle(HandlerInput input) {
-        String speechText = "Welcome to the Alexa Skills Kit sample. Please tell me your favorite color by saying, my favorite color is red";
+        String speechText = "Welcome to the Alexa Skills Kit sample. Please tell me your favorite color by saying, my favorite color is red. " +
```

Comment:
Are those utterance changes reviewed by COPs?

Reply:
@tianrenz , are you asking adding `"Your options are green, blue, purple, red, orange and yellow."`? This [ticket](https://tt.amazon.com/0155095586) was from COPS, and they did recommend to validate user provided color. 

I added `options` sentence in `MyColorIsIntentHandler`, I think it makes more sense to add it in `LaunchRequest` as well.